testsuite/microprofile-tck/pom.xml

```diff
@@ -55,8 +55,9 @@
     </dependency>
 
     <dependency>
-      <groupId>org.wildfly</groupId>
-      <artifactId>wildfly-microprofile-config-smallrye</artifactId>
+      <groupId>io.smallrye</groupId>
+      <artifactId>smallrye-config</artifactId>
+      <version>1.3.6</version>
```

Comment:
Out of curiosity, why is this needed?
In any case, we should also fix the resteasy-dependencies-bom/pom.xml accordingly.

Reply:
Implementation of rest client depends on config :(